lib/fluent/plugin/out_graphite.rb

```diff
@@ -49,19 +49,26 @@ def configure(conf)
     end
     # How many times to retry the call if timeout raised
     @max_retries ||= 3
+
+    if @flush_interval < 10
+      log.info("flush_interval less than 10s is not allowed and overwritten to 10s")
+      @flush_interval = 10
```

Comment:
> 10 seconds is just a default value for the highest-resolution retention period, and it's configurable, right?

Yes. Users can set arbitrary value greater than 10 seconds with `flush_interval` parameter in configuration.



> Don't you have any other solution to solve the problem...?

An another approach is using this plugin fluent-plugin-bufferize together.
This approach is also acceptable for me.
But it remains some pitfall.